src/lib.cairo

```diff
@@ -38,31 +38,64 @@ mod Account {
     use starknet::get_contract_address;
     use starknet::ContractAddress;
     use zeroable::Zeroable;
+    use starknet::contract_address::contract_address_const;
 
-    use arcadeAccount::account::interface;
-    use arcadeAccount::introspection::interface::ISRC5;
-    use arcadeAccount::introspection::interface::ISRC5Camel;
-    use arcadeAccount::introspection::src5::SRC5;
+    use arcade_account::account::interface;
+    use arcade_account::introspection::interface::ISRC5;
+    use arcade_account::introspection::interface::ISRC5Camel;
+    use arcade_account::introspection::src5::SRC5;
 
     use super::Call;
     use super::QUERY_VERSION;
     use super::TRANSACTION_VERSION;
 
-    use arcadeAccount::utils::{selectors, contracts};
+    use arcade_account::utils::{selectors, contracts};
+
+    const TRUE: felt252 = 1;
+    const FALSE: felt252 = 0;
```

Comment:
can we use the built-in bool?

Reply:
Sure. I was just thinking about gas efficiency but I guess it's over optimization

Reply:
Updated